package.json

```diff
@@ -23,10 +23,11 @@
     "devDependencies": {
         "@dhis2/cli-app-scripts": "^10.4.1",
         "@dhis2/cli-style": "^10.4.1",
-        "@dhis2/cypress-commands": "^9.0.2",
-        "@dhis2/cypress-plugins": "^9.0.2",
-        "cypress": "^9.7.0",
-        "cypress-cucumber-preprocessor": "^4.3.1",
+        "@dhis2/cypress-commands": "^10.0.6",
+        "@dhis2/cypress-plugins": "^10.0.1",
+        "cypress": "^12.7.0",
```

Comment:
Ah I guess this is straight from the guide. We should be able to use the latest version (13 point something), I don't think anything related to what we do at dhis2 has changed